graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH1.kt

```diff
@@ -0,0 +1,120 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+@BEH(GraphBehaviours.GetCycle, GraphBehaviours.CreateDirectedGraph, """
+    Check cycles exist in directed graph
+""")
+class BEH1 : GraphScenario() {
+
+    @Eac("A graph has one cycle")
```

Comment:
:no_entry_sign: This is not an AC (and nor are the others in this file).

Here an AC might read... `When a graph contains a cycle, gives true`

By expressing your ACs in this form you will also likely see that some of your scenarios are not in fact ACs at all as they do not consider a unique situation.

When you have a list of ACs it should read like a list of ACs that a PO would give an engineer before implementation.

I have not gone through your ACs in detail as it would probably be good for you to review given the above comment first.